cli/cli.go

```diff
@@ -59,6 +59,11 @@ func init() {
 	fs.StringVar(&defaults.RedisURL, "redis_url", redisDefault, "")
 	fs.StringVar(&defaults.RedisChannel, "redis_channel", "__anycable__", "")
 
+	fs.BoolVar(&defaults.RedisSentinelEnabled, "redis_sentinel_enabled", false, "")
```

Comment:
Can we simplify the configuration by re-using `redis_url` for master name and password?
And also make `redis_sentinel_enabled` by checking whether the list of sentinels is provided or not?

Thus, only a single new parameter would be required.

P.S. That's the way Ruby client handles this: https://bugs.ruby-lang.org/versions/36